src/elements/content-sidebar/activity-feed/task-form/TaskError.js

```diff
@@ -9,23 +9,23 @@ import getProp from 'lodash/get';
 
 import messages from './messages';
 import apiMessages from '../../../../api/messages';
-import { TASK_EDIT_MODE_EDIT } from '../../../../constants';
+import { TASK_EDIT_MODE_EDIT, TASK_MAX_GROUP_ASSIGNEES } from '../../../../constants';
 import InlineNotice from '../../../../components/inline-notice/InlineNotice';
 
 import type { TaskType, TaskEditMode } from '../../../../common/types/tasks';
 
 type Props = {
     editMode?: TaskEditMode,
-    error?: { status: number },
+    error?: { status: number }, // TODO: update to ElementXhrError once API supports it
     taskType: TaskType,
 };
-
 const TaskError = ({ editMode, error, taskType }: Props) => {
     const isEditMode = editMode === TASK_EDIT_MODE_EDIT;
     const isForbiddenErrorOnEdit = getProp(error, 'status') === 403 && isEditMode;
+    const taskGroupExceedsError = getProp(error, 'code') === 'group_exceeds_limit';
```

Comment:
should you use the constant `ERROR_CODE_GROUP_EXCEEDS_LIMIT` here?

Reply:
good catch. will update